R/table-format.R

```diff
@@ -153,18 +153,32 @@ pad_prop <- function(x, digits, fmt_small = TRUE, keep_zero = FALSE,
   digits <- check_pos_int(digits)
   output <- check_output(output)
   new_x <- fmt_prop(x, digits = digits, fmt_small = fmt_small,
-                    keep_zero = keep_zero)
+                    keep_zero = keep_zero, output = output)
   new_x[is.na(new_x)] <- "NA"
 
-  if (any(stringr::str_detect(new_x, "^<|^>")) &
-      !all(stringr::str_detect(new_x, "^<|^>"))) {
-    pad <- ifelse(output == "latex", 4, 3)
-    new_x <- dplyr::case_when(stringr::str_detect(new_x, "^<|^>") ~
-                                paste0(new_x, paste(rep("\\ ", pad),
-                                                    collapse = "")),
-                              TRUE ~ new_x)
+  if(is_html_output()) {
+    if ((any(stringr::str_detect(new_x, "lt")) |
+         any(stringr::str_detect(new_x, "gt"))) &
+        !(all(stringr::str_detect(new_x, "lt") |
+              stringr::str_detect(new_x, "gt")))) {
+      pad <- ifelse(output == "latex", 4, 3)
```

Comment:
This is currently in an `if (is_html_output())`, so this should only be for HTML, no?

I would recommend moving this line of code outside of the HTML/LaTeX if statements, then you just have the `pad` variable defined when you go into the if.